recipes/r-sinew/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = '0.3.8' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sinew
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sinew_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sinew/sinew_{{ version }}.tar.gz
+  sha256: 81b993655dfe2283513fff793d442d5bfe7c5e7ab514666d325bdb571f3ffdbd
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-cli
+    - r-crayon
+    - r-miniui
+    - r-rematch2
+    - r-rstudioapi
+    - r-shiny
+    - r-sos
+    - r-stringi
+    - r-yaml
+  run:
+    - r-base
+    - r-cli
+    - r-crayon
+    - r-miniui
+    - r-rematch2
+    - r-rstudioapi
+    - r-shiny
+    - r-sos
+    - r-stringi
+    - r-yaml
+
+test:
+  commands:
+    - $R -e "library('sinew')"           # [not win]
+    - "\"%R%\" -e \"library('sinew')\""  # [win]
+
+about:
+  home: https://github.com/metrumresearchgroup/sinew
+  license: GPL-2.0-only OR GPL-3.0-only
+  summary: Create 'roxygen2' skeleton populated with information scraped from the within the
+    function script. Also creates field entries for imports in the 'DESCRIPTION' and
+    import in the 'NAMESPACE' files. Can be run from the R console or through the 'RStudio'
+    'addin' menu.
+  license_family: GPL3
```

Comment:
Maybe the family should be only GPL since the license is GPL2 or GPL3?

Reply:
Thanks for the comment, fixed in 1eaed9c.